src/azure-cli-core/azure/cli/core/_util.py

```diff
@@ -167,3 +167,10 @@ def b64encode(s):
         return encoded
     else:
         return encoded.decode('latin-1')
+
+
+def random_string(length=16, force_lower=False):
+    from string import ascii_letters, digits, ascii_lowercase
+    from random import choice
+    choice_set = ascii_lowercase + digits if force_lower else ascii_letters + digits
+    return ''.join([choice(choice_set) for n in range(length)])  # pylint: disable=unused-variable
```

Comment:
Since the variable is unused, you can replace `n` with `_`. This should remove the pylint warning and clearly shows that you don't use `n` either.